app/controllers/repositories_controller.rb

```diff
@@ -388,7 +388,13 @@ def to_csv(rows, column_ids)
                      else
                        cell = row.repository_cells
                                  .find_by(repository_column_id: c_id)
-                       cell ? cell.value.data : nil
+                       if !cell
+                         nil
+                       elsif cell.value_type == 'RepositoryListValue'
+                         cell.value.repository_list_item.data
+                       else
+                         cell.value.repository_list_item.data
```

Comment:
May be  'cell.value.data' ?

Reply:
Ah yes, of course, a silly copy-paste mistake...